src/Roslyn.Diagnostics.Analyzers/Core/SpecifyDiagnosticDescriptorCustomTags.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Roslyn.Diagnostics.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class SpecifyDiagnosticDescriptorCustomTags : DiagnosticAnalyzer
```

Comment:
Let me know if you prefer another name.

Reply:
You don't need a new analyzer. You can just extend https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzer.cs, which already has a very similar rule for missing helpLinkUri argument.

Reply:
I shall pay more attention to the existing analyzers... Closing this PR and working on extending the other analyzer.